src/app.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const pathname = normalizedURL.pathname;
+
+  if (!pathname.startsWith('/file')) {
+    res.end('The pathname should starts with "/file"');
+
+    return;
+  }
+
+  const fileName = pathname.slice(1).split('/').slice(1).join('/')
```

Comment:
You could do like this:
```
pathname.split('/').slice(2).join('/')
```
It will be the same but shorter by one method.

Reply:
Oh, you're dead right))